examples/mockups/pricing.html

```diff
@@ -0,0 +1,786 @@
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">Enhanced full-text search</div>
+                    </div>
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">API access</div>
+                    </div>
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">Unlimited projects & users</div>
+                    </div>
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">Private Versions</div>
+                    </div>
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">Live PR preview</div>
+                      <div class="description">Live preview of all pull requests</div>
+                    </div>
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">Email build failure notifications</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">2 day support response time</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">4 Concurrent builds</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Custom domain</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Content embedding</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Search term analytics</div>
+                      <div class="description">30 days</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Pageview analytics</div>
+                      <div class="description">30 days</div>
+                    </div>
+                  </div>
+                </div>
+
+              </div>
+
+            </div>
+
+            <div class="extra content">
+              <span class="left floated share">
+                <i class="icon share alternate"></i>
+                Share
```

Comment:
This is not a _must have_ feature but if might be cool to have, that said, I've thought about some options:

* Link to this pricing page block with the card highlighted
(less interesting option)
* Link to the next page - the same as in the "learn more" link - where we'll have the community vs corporate separated with more details on each
(if we decide do go that way)
* Link to a pricing page unique for this plan
(if there is one)

Reply:
We discussed this one a little bit already, but for now I don't think we're planning on a more detailed page for our plans yet, besides maybe a page dedicated to commercial, enterprise level contracts.

So that probably only leaves the first option, currently at least. Highlighting the card is an option, we could also mute/disable the other plan cards that don't match. I could see us wanting this at some point -- the ability to link into our pricing page by plan from our docs or dashboard UI for example.

Would this be part of the initial implementation, or a new feature we should explore later?

Reply:
We can explore this later. There's not much else depending on a decision here.

For the 1st option, we just need to add a class to highlight the card and then add some behaviour.

The other options lead us to a more in-depth design discussion of the code here and the use of `imports` for each plan that would be coded in a separate file so it can be imported to different places as needed...

We can start implementing this follow up idea, as it's useful anyway, or can also move the code to separate files at a later stage.

Reply:
Just noticed that #11 is related to options mentioned here but with a different approach: separating into different files/references by feature and not by plan.